unit-tests/live/frames/test-pipeline-start-stop.py

```diff
@@ -0,0 +1,41 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+# test:device D400*
```

Comment:
Why only D400?

Reply:
Other are about to be deprecated, no need to add run time for them